Rework bdr.drop_node (#6056)
EnterpriseDB/docs: product_docs/docs/pgd/5/reference/functions-internal.mdx

### `bdr.drop_node`

Drops a node's metadata.

This function removes the metadata for a given node from the local
database. The node can be either:

-   The local node, in which case it removes all the node metadata, including information about remote nodes.
-   A remote node, in which case it only removes metadata for that specific node.

!!! Important When to use `bdr.drop_node()`
You don't have to use `bdr.drop_node` to remove a node from a PGD group if you are reusing the node name. This is because PGD can reuse existing node names providing the node being named is in a `PARTED` state.

Generally, you should use [`bdr.part_node`](/pgd/latest/reference/nodes-management-interfaces#bdrpart_node) to remove a node from a PGD group (which sets the node to `PARTED` state). You should only use `bdr.drop_node` function when:

> * if you need to remove metadata of a `PARTED` node under the instruction of EDB Technical Support

Author: ```suggestion
* you need to remove metadata of a `PARTED` node under the instruction of EDB Technical Support
```

### `bdr.drop_node`

Drops a node's metadata.

This function removes the metadata for a given node from the local
database. The node can be either:

-   The local node, in which case it removes all the node metadata, including information about remote nodes.
-   A remote node, in which case it only removes metadata for that specific node.

!!! Important When to use `bdr.drop_node()`

> Because PGD can reuse existing node names providing the node name belongs to a node in a `PARTED` state, you should not use `bdr.drop_node` to remove the original node metadata.

Reviewer: How about this?
```suggestion
Because PGD can reuse existing node names, you should not use `bdr.drop_node` to remove the original node metadata, unless the node name belongs to a node in a `PARTED` state.
```

Author: No, that's the wrong way around. 

```suggestion
PGD can reuse existing node names providing the node name belongs to a node in a `PARTED` state. To reuse a node name, use `bdr.part_node()` to remove the original node from the cluster. You should not use `bdr.drop_node` to remove the original node metadata. 
```
